dask_expr/collection.py

```diff
@@ -803,3 +804,47 @@ def read_parquet(
             kwargs=kwargs,
         )
     )
+
+
+def concat(
+    dfs,
+    axis=0,
+    join="outer",
+    ignore_unknown_divisions=False,
+    ignore_order=False,
+    **kwargs,
+):
+    if axis == 1:
+        # TODO: implement
+        raise NotImplementedError
+    if ignore_unknown_divisions:
+        # TODO: implement
+        raise NotImplementedError
+
+    if not isinstance(dfs, list):
+        raise TypeError("dfs must be a list of DataFrames/Series objects")
+    if len(dfs) == 0:
+        raise ValueError("No objects to concatenate")
+    if len(dfs) == 1:
+        return dfs[0]
+
+    if join not in ("inner", "outer"):
+        raise ValueError("'join' must be 'inner' or 'outer'")
+
+    if axis == 1:
```

Comment:
You've got `if axis == 1` a couple times here.  I guess we have an optimization that will be useful once we turn on this capability?

Reply:
I think they are all gone for now. Will try to integrate this better when implementing axis=1